tests/generator/generator.cpp

```diff
@@ -203,7 +203,7 @@ TEST(Generator, variableInitializedTwice)
     EXPECT_EQ(size_t(0), parser->issueCount());
 
     const std::vector<std::string> expectedIsues = {
-        "Variable 'x' in component 'sub' of model 'variable_initialized_twice' and variable 'x' in component 'main' of model 'variable_initialized_twice' are equivalent and cannot therefore both be initialised.",
+        "Variable 'x' in component 'sub' and variable 'x' in component 'main' are equivalent and cannot therefore both be initialised.",
```

Comment:
I wonder whether our messages should talk about things that reflect the spec - ie: "connected" vs. "equivalent" or things that reflect our function names (`addEquivalence` etc?)

Reply:
We are dealing with libCellML here, not the specs. So, since our API uses `addEquivalence()`, etc., I would stick to "equivalent".